commit d7580215c9e39eddc33196b32b05219fcea707fd
Author: Steven Peters <scpeters@osrfoundation.org>
Date:   Fri Jan 15 16:29:37 2016 -0800

    gazebo_ros_utils.h: include gazebo_config.h
    
    Make sure to include gazebo_config.h,
    which defines the GAZEBO_MAJOR_VERSION macro

diff --git a/gazebo_plugins/include/gazebo_plugins/gazebo_ros_utils.h b/gazebo_plugins/include/gazebo_plugins/gazebo_ros_utils.h
index 6cdc4a8..6b016b8 100644
--- a/gazebo_plugins/include/gazebo_plugins/gazebo_ros_utils.h
+++ b/gazebo_plugins/include/gazebo_plugins/gazebo_ros_utils.h
@@ -39,6 +39,7 @@
 #include <gazebo/common/common.hh>
 #include <gazebo/physics/physics.hh>
 #include <gazebo/sensors/Sensor.hh>
+#include <gazebo/gazebo_config.h>
 #include <ros/ros.h>
 
 #ifndef GAZEBO_SENSORS_USING_DYNAMIC_POINTER_CAST
